### PR TITLE
Improve UI for package voting, Reverse versions in package search

### PR DESCRIPTION
**Pull request by pboyer**
### Issue

The UI for package downvoting was often confused with and "expand" button.  I've made it less ambiguous with the help of @lillismith.

I've also reversed the order of the versions as shown in the expanded package search.

Along the way, I did some minor code cleanup, namely to remove the `PackageManagerClient` reference from `PackageManagerSearchElement`.

screenshot

### Reviewer

- [x] @ikeough

### CODE REVIEW

File: src/DynamoCoreWpf/ViewModels/PackageManager/PackageManagerSearchElementViewModel.cs [hunk 1]
```diff
@@ -65,7 +64,7 @@ private static void GoToUrl(string url)
             {
                 return
                     Model.Header.versions.Select(
-                        x => new Tuple<PackageVersion, DelegateCommand>(x, new DelegateCommand(() => OnRequestDownload(x)))).ToList();
+                        x => new Tuple<PackageVersion, DelegateCommand>(x, new DelegateCommand(() => OnRequestDownload(x)))).Reverse().ToList();
```
Comment (pboyer) [inline src/DynamoCoreWpf/ViewModels/PackageManager/PackageManagerSearchElementViewModel.cs]: This line fixes MAGN-7875.

File: src/DynamoCoreWpf/Views/PackageManager/PackageManagerSearchView.xaml [hunk 2]
```diff
@@ -203,23 +202,23 @@
...
+                                                <StackPanel Orientation="Horizontal" Margin="5" Width="50">
+                                                    <TextBlock FontSize="10" FontFamily="Segoe UI Symbol" Text="&#xE118;   " Foreground="Gray" />
                                                     <TextBlock FontSize="10" Text="{Binding Path= Model.Downloads}" Foreground="Gray"  />
                                                 </StackPanel>
 
-                                                <StackPanel Orientation="Horizontal" Margin="5" Width="75">
-                                                    <TextBlock FontSize="10" FontWeight="Bold" Text="◷   " Foreground="Gray" />
+                                                <StackPanel Orientation="Horizontal" Margin="5" Width="80">
+                                                    <TextBlock FontSize="10" FontFamily="Segoe UI Symbol" Text="&#xE121;   " Foreground="Gray" />
                                                     <TextBlock FontSize="10" Text="{Binding Path= Model.LatestVersionCreated, Converter={StaticResource PrettyDateConverter}}" Foreground="Gray"  />
```
Comment (pboyer) [inline src/DynamoCoreWpf/Views/PackageManager/PackageManagerSearchView.xaml]: I used "Segoe UI Symbol" available on Windows 7 on.

File: src/DynamoCoreWpf/Views/PackageManager/PackageManagerSearchView.xaml [hunk 3]
```diff
@@ -203,23 +202,23 @@
...
-                                                <StackPanel Orientation="Horizontal" Margin="5" Width="75">
+                                                <StackPanel Orientation="Horizontal" Margin="5" Width="60">
                                                     <TextBlock FontSize="10" FontWeight="Bold" Text="⚑   " Foreground="Gray" />
                                                     <TextBlock FontSize="10" Text="{Binding Path= Model.LatestVersion}" Foreground="Gray"  />
                                                 </StackPanel>
 
-                                                <StackPanel Orientation="Horizontal" Margin="5" Width="75">
-                                                    <TextBlock FontSize="10" FontWeight="Bold" Text="▼   " Foreground="Gray" />
+                                                <StackPanel Orientation="Horizontal" Margin="5" Width="50">
+                                                    <TextBlock FontSize="10" FontFamily="Segoe UI Symbol" Text="&#xE118;   " Foreground="Gray" />
```
Comment (ikeough) [inline src/DynamoCoreWpf/Views/PackageManager/PackageManagerSearchView.xaml]: Ha! That's awesome.

**Comment by ikeough** [comment]
LGTM, with two small comments.

**Pull request merged**